go.mod

```diff
@@ -31,8 +31,8 @@ require (
 	github.com/hashicorp/golang-lru v0.5.3
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/karrick/godirwalk v1.10.12 // indirect
-	github.com/kubeedge/beehive v0.0.0
-	github.com/kubeedge/viaduct v0.0.0
+	github.com/kubeedge/beehive v0.0.0-20201125122335-cd19bca6e436
```

Comment:
We can't indicate beehive and viaduct to a specific verison here, since they are replaced with `./staging/src/github.com/kubeedge/beehive` and `./staging/src/github.com/kubeedge/viaduct` in the `replace ()` section.
You would have to modify the version tag in your own go.mod where you import the kubeedge/kubeedge repo.

Reply:
This the way i have chosen